MediaProcessor/tests/ConfigFileCreator.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CONFIGFILECREATOR_H
+#define CONFIGFILECREATOR_H
+
+#include <filesystem>
+#include <nlohmann/json.hpp>
+#include <stdexcept>
+
+namespace MediaProcessor::UnitTesting {
```

Comment:
bit of a nitpick, let's rename the NS to `MediaProcessor::Tests`

Reply:
ok